andela/nazgul-rc
# #157194656 Admin should be able to cancel order

#### What does this PR do?
It allows an admin cancel an order

#### Description of Task to be completed?
N/A
#### How should this be manually tested?
clone this repository 
`cd ` into the project directory using your terminal 
if you have installed all the projected dependencies 
>- run the command `reaction` to start the application
however if you have not installed the project dependencies 
>- run the command `meteor npm install` and `npm install`
>- after the above run `reaction`
when the application starts 
navigate to the web address localhost:3000/
login as an admin with the following credentials
>- `admin@localhost`
>- `r3@cti0n`
- after login toggle the button at the top left hand side of the page to switch off edit mode
- then click on the `Basic Reaction Product` card to start making an order
- on the `example-product page` add to `cart` and then proceed to checkout
- On the checkout page fill out the required information
- on the form where you are required to `Add payment to complete order`
- fill the form to complete payment preferable with a real debit card 
- click on the   `complete your order` button to complete your checkout
- navigate to the `orders` panel to see the order 
- click on the blue arrow to see the invoice panel
- on the invoice panel are two buttons `Approve` and `Cancel`
- click `cancel` to cancel the order
- the customer would get an in-app notification that their order has been canceled.
- the customer would get a mail notification that their order has been canceled 


#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/157194656

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-05-12 at 10 12 31 am" src="https://user-images.githubusercontent.com/24682326/39956153-22e175aa-55d3-11e8-98ed-72a56993873a.png">
<img width="1440" alt="screen shot 2018-05-12 at 10 12 53 am" src="https://user-images.githubusercontent.com/24682326/39956154-2309b344-55d3-11e8-8a4a-4e69cee5907b.png">
<img width="1440" alt="screen shot 2018-05-12 at 10 13 04 am" src="https://user-images.githubusercontent.com/24682326/39956155-23c70048-55d3-11e8-860a-7946167a9b3b.png">
<img width="804" alt="screen shot 2018-05-12 at 10 18 32 am" src="https://user-images.githubusercontent.com/24682326/39956156-23eb23d8-55d3-11e8-9c9c-b162e07c704c.png">
<img width="1438" alt="screen shot 2018-05-17 at 2 20 55 pm" src="https://user-images.githubusercontent.com/24682326/40179973-9b6ad56c-59dd-11e8-97e0-40baa206248f.png">


#### Questions:
N/A
